integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -32,4 +32,17 @@ public void webhookSiteContainsRequest(String requestText, int timeoutMinutes) {
                                 requestTextWithoutPlaceholders)
                         .anyMatch(requestContent -> requestContent.contains(requestTextWithoutPlaceholders)));
     }
+
+    @Then("^Webhook site does not contains request with text \"([^\"]*)\" within (\\d+) (?:minute|minutes)$")
```

Comment:
The minute long timeout is quite long, IMHO it is enough to have 10 second timeout